app/chewy/stafftools_index.rb

```diff
@@ -0,0 +1,142 @@
+class StafftoolsIndex < Chewy::Index
+  define_type Assignment do
```

Comment:
@johndbritton I've added all of the models as objects for the `StafftoolsIndex` let me know if there other attributes that you think would be useful here.